stable/insights-agent/templates/token-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
A lot of people (including Fairwinds) don't like create secrets using Helm due to how it stores those values in the cluster.  I would like to see this chart at least provide the option to specify a pre-existing secret name.

Reply:
Makes sense. Done!